### DISCUSSION TITLE
Doesn´t work

### DISCUSSION THREAD

**Discussion by Robthelion** [rxhanson/RectanglePro-Community#118] [Q&A]
I´ve downloaded the trial version. It asks to open system preferences and chek it for availability. It doesn´t work for me on mac 12.6
Any help????

**Comment by rxhanson** [comment]
Thanks for trying it. After checking the box in System Preferences, what happens? What do you see in the menu bar menu?

**Reply by Robthelion** [reply]
The application doesn´t authorize me.

**Reply by Robthelion** [reply]
![por defecto 2022-09-22 a las 21 05 06](https://user-images.githubusercontent.com/114183716/191830700-1f13ec44-9888-4f72-9626-27f60cfd8fa1.jpg)

**Reply by Robthelion** [reply]
When I check and close the app doesn´t work

**Reply by rxhanson** [reply]
When this happens, it means that macOS is still telling the app that it doesn't have permission, even though it does. Typically this is fixed by quitting the app and starting it again, or rebooting your Mac (or sometimes just logging out then back in).

**Reply by Robthelion** [reply]
I´ll try it but I quitted several times without rebooting. Many thanks!!!!!!
